### PR TITLE
Add openseadragon w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@cdnjs/intern2 Please help me review this PR, thanks.
Pull request for issue: #8811 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/openseadragon/openseadragon
 * Official website (optional, not the repository): https://openseadragon.github.io/
 * NPM package url (optional): https://www.npmjs.com/package/openseadragon
 * License and its reference: BSD-3-Clause, https://openseadragon.github.io/license/
 * GitHub / Bitbucket popularity (required):
   Watch 79
   Star 1,123
   Fork 256
 * NPM download stats (optional):
   119 downloads in the last day
   588 downloads in the last week
   2,484 downloads in the last month

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Let's give @extend1994 a try.

File: ajax/libs/openseadragon/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "osm",
+    "tms"
+  ],
+  "homepage": "https://openseadragon.github.io/",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openseadragon/openseadragon.git"
+  },
+  "author": {
```
Comment (maruilian11) [inline ajax/libs/openseadragon/package.json]: It is added by myself. I traced the commit contributor, and all members of openseadragon contribute to this project.
https://github.com/openseadragon/openseadragon/graphs/contributors

Comment (maruilian11) [inline ajax/libs/openseadragon/package.json]: Next time, you can try to trace yourself first. 👍

Comment (PeterDaveHello) [inline ajax/libs/openseadragon/package.json]: I think we don't need to add author info by our own, there are lots of things need to be done first.

**Comment by maruilian11** [comment]
@extend1994 I just removed the `author` field, and I think the current commit is just fine, thanks for your advice. Please help me review this PR, thank you.

**Comment by maruilian11** [comment]
@extend1994 If you are the last reviewer, please help ping @PeterDaveHello. BTW, could you please help rebase the branch w/ latest master branch if you not review in 24h. Thank you.

**Comment by extend1994** [comment]
@maruilian11 Oh I missed that, sorry! Then if I am not the latest reviewer and did not review in 24 hours, do I also need to rebase the branch?

**Comment by maruilian11** [comment]
@extend1994 If the PR still need others to review, it's ok to not rebase it. Thanks!

**Comment by PeterDaveHello** [comment]
Should better rebase it as I'm not in the review queue.

**Pull request merged**